test/sparse/sparse.jl

```diff
@@ -1807,7 +1807,7 @@ end
     @test String(take!(io)) ==  string("6×3 SparseMatrixCSC{Float64,$Int} with 18 stored entries:\n  [1, 1]",
                                        "  =  1.0\n  [2, 1]  =  1.0\n  ⋮\n  [5, 3]  =  1.0\n  [6, 3]  =  1.0")
 
-    ioc = IOContext(io, :displaysize => (9, 80))
+    ioc = IOContext(io, displaysize = (9, 80))
```

Comment:
As of today, it seems to be faster and to allocate less to use the `Pair` version rather than keywords; so this commit was more for code-style rather than for reducing allocations right?